examples/analog_hamiltonian_simulation/09_Noisy_quantum_dynamics_for_Rydberg_atom_arrays.ipynb

```diff
@@ -0,0 +1,1698 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.            n_height = int(float(height) // (atom_separation + patch_separation))
</pre><p><code>height</code> and <code>width</code> are already float  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>